pom.xml

```diff
@@ -198,34 +200,35 @@
           </execution>
         </executions>
       </plugin>
-<!--
-      See https://jira.apache.org/jira/browse/HUDI-304
       <plugin>
         <groupId>com.diffplug.spotless</groupId>
         <artifactId>spotless-maven-plugin</artifactId>
-        <version>1.24.3</version>
+        <version>${spotless.version}</version>
         <configuration>
           <java>
-            <eclipse>
-              <file>${main.basedir}/style/eclipse-java-google-style.xml</file>
-              <version>4.10.0</version>
-            </eclipse>
-          </java>
-          <scala>
+            <googleJavaFormat>
```

Comment:
what does Flink use? I think it does `AOSP` ?  the wraps are lot more nicer, the one arg per line, really makes it hard to get a good chunk of code read in one screen.

Reply:
e.g https://github.com/apache/flink/blob/master/flink-clients/src/main/java/org/apache/flink/client/ClientUtils.java